documentation/api/io.strimzi.api.kafka.model.KafkaBridgeConsumerSpec.adoc

```diff
@@ -6,24 +6,22 @@ The values can be one of the following JSON types:
 * Number
 * Boolean
 
-You can specify and configure the options listed in the {ApacheKafkaConsumerConfig} with the exception of those options which are managed directly by Strimzi.
-Specifically, all configuration options with keys equal to or starting with one of the following strings are forbidden:
+*Exceptions*
 
-* `ssl.`
-* `sasl.`
-* `security.`
-* `bootstrap.servers`
-* `group.id`
```

Comment:
I would probably keep this -> I think it helps some of the users to see the exact options. ... at least unless the list is somewhere else as well.

Reply:
I agree with Jakub.

Reply:
Th exact options are in the table description for `config`, hence the sentence that replaces the list (_See the description of the `config` property for a list of prefixes for options you cannot change._) Not sure why we want to repeat the list here?

![image](https://user-images.githubusercontent.com/47596553/234322728-f7e59830-e046-4382-aafc-4781cf2c1115.png)

Reply:
I don't know - to me it seems a bit more readable as an exact list. From my point of view, the list you added instead of it has the same information in a less useful form. So I do not really follow the reasoning for this.

Reply:
It's certainly more readable, but we're just maintaining the same list in two places. I did wonder if we should change the format in the table (@Description) or remove them from the tables, but both options would be reflected in the YAML etc.

Anyway, I've added the lists, with some updates too